gpiozero/internal_devices.py

```diff
@@ -461,8 +461,7 @@ class DiskUsage(InternalDevice):
     """
     def __init__(self, filesystem='/', threshold=90.0, pin_factory=None):
         super(DiskUsage, self).__init__(pin_factory=pin_factory)
-        if not os.path.ismount(filesystem):
-            raise ValueError('invalid filesystem')
+        os.statvfs(filesystem)
```

Comment:
Might be worth adding a comment in here explaining that this is just checking that the `filesystem` exists, which is why the return value from statvfs is being thrown away? Looks mighty weird otherwise ;)

Reply:
Good point - and yes, that's exactly what it's doing :)